src/2982R1_std_quantity_as_a_numeric_type.md

```diff
@@ -2557,7 +2558,7 @@ Doing so has two important benefits:
    the thing being returned there.
 2. Such a concept constrains the type returned from the function. This means that it works as
    a unit test to verify if our function actually performs what it is supposed to do. If there is
-   an error in quantity equations, we will learn about it right away.
+   an error in quantity expressions, we will learn about it right away.
```

Comment:
This is incorrect. The correct term is "quantity equation" (https://jcgm.bipm.org/vim/en/1.22.html),

Reply:
Hmm... Yeah, I think that makes sense.
The thing is that the equal sign of the equation is invisible in code,
so I thought it was an expression (<https://en.wikipedia.org/wiki/Expression_(mathematics)>).
In the case of `struct second : ... { } second;`,
the "assignment" of the expression in the base type list to the name `second` makes an equation.